mycroft/skills/main.py

```diff
@@ -438,11 +438,17 @@ def load_priority(self):
                         continue
             self._load_or_reload_skill(skill.path)
 
+    def remove_git_locks(self):
+        for i in glob(join(self.msm.skills_dir, '*/.git/index.lock')):
+            LOG.warning('Found and removed git lock file: ' + i)
+            os.remove(i)
+
     def run(self):
         """ Load skills and update periodically from disk and internet """
 
         # Load priority skills first, in order (very first time this will
         # occur before MSM has run)
```

Comment:
Let's clean this up -- the comment got left behind and makes absolutely no sense anymore.  The code was originally this:
```python
        # Load priority skills first, in order (very first time this will
        # occur before MSM has run)
        self.load_skill_list(PRIORITY_SKILLS)
```

So, that comment is no confusing, not helpful.  Axe it.

Additionally, I think we need to explain the reasoning for the code that removes .git lock files.   In the future, somebody is going to look at this and ask: "Why would someone do this? .git/index.lock is there for a reason!"

So add a comment that describes the reasoning behind the decision to override the locking mechanism.

Reply:
Fixed!